lib/engine/game/g_1848/step/buy_train.rb

```diff
@@ -22,6 +22,17 @@ def actions(entity)
             end
           end
 
+          def can_buy_train?(entity = nil, _shell = nil)
+            return true if only_owns_ghan?(entity)
```

Comment:
This change is so the trainless corp can continue trying to buy trains after it buys a 2E